wesnoth-units.po

```diff
@@ -6285,11 +6279,12 @@ msgstr ""
 msgid "Black Stallion"
 msgstr "Černý hřebec"
 
+# Nedokážu do toho vsunout nějakou větší slovní hříčku i v češtině.
 #. [female]
 #. Female variant of the Black Horse or Black Stallion; in English this is a pun on the word "Mare"
 #: data/core/units/monsters/Horse_Black.cfg:91
 msgid "female^Black Nightmare"
-msgstr "Černá můra"
+msgstr "Černá kobyla"
```

Comment:
?
```suggestion
msgstr "Černá Máry"
```

Reply:
No úplně to není asi ono, hlavně z hlediska toho, že to indikuje pozmenování jedince namísto pojmenování druhu.

Reply:
Asi jako pražskej Pepík? No, nic vám necpu.

Reply:
Černá máry je kulturně spojená např. s tramvají, s černošskou vůdkyní nebo s negativní přezdívkou bachaře. Bohužel tam nefiguruje žádný kůň ani třeba ve formě konského spřežení.
Jako poeticky to zní pěkně, ale významově i ta můra víc zapadá.
Černá Máry by bylo dobré jméno pro konkrétní černou kobylu, co by vystupovala v nějakém tažení, ale obecně...